orc8r/lib/go/analytics/calculations/user_state.go

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
I don't think this code should be in orc8r/lib. Typically we've restricted this library to be for functionality explicitly used by both cloud and gateway such that gateway's don't have to import the entire cloud go mod. For example, the builder protos for mconfig sit at `magma/orc8r/cloud/go/services/configurator/protos`. I can't see a usecase for gateways needing this. I think it was fine where it was.